xero-payroll-au.yaml

```diff
@@ -4999,7 +5056,6 @@ components:
       enum:
       - AUSTRALIANRESIDENT
       - FOREIGNRESIDENT  
-      - WORKINGHOLIDAYMAKER
```

Comment:
Is this why this is considered a breaking change, the removal an enum option?

Reply:
It sounds like WORKINGHOLIDAYMAKER is still supported if the organisation hasn't yet enabled STP Phase 2. Should this be included until all AU orgs have converted to STP Phase 2? Otherwise we will have marshalling errors.

Reply:
Resolved by adding `WORKINGHOLIDAYMAKER` back ✔️